manifest.datestamps.json

```diff
@@ -1,40 +1,45 @@
       "3.5-ltsc2019-Runtime-DateStamp": "$(RuntimeReleaseDateStamp)",
       "3.5-ltsc2019-Aspnet-DateStamp": "$(AspnetReleaseDateStamp)",
       "3.5-ltsc2019-Sdk-DateStamp": "$(SdkReleaseDateStamp)",
       "3.5-ltsc2022-Runtime-DateStamp": "$(RuntimeReleaseDateStamp)",
       "3.5-ltsc2022-Aspnet-DateStamp": "$(AspnetReleaseDateStamp)",
       "3.5-ltsc2022-Sdk-DateStamp": "$(SdkReleaseDateStamp)",
-      "4.6.2-ltsc2016-Runtime-DateStamp": "$(RuntimeReleaseDateStamp)",
-      "4.6.2-ltsc2016-Aspnet-DateStamp": "$(AspnetReleaseDateStamp)",
-      "4.6.2-ltsc2016-Wcf-DateStamp": "$(WcfReleaseDateStamp)",
-      "4.7-ltsc2016-Runtime-DateStamp": "$(RuntimeReleaseDateStamp)",
-      "4.7-ltsc2016-Aspnet-DateStamp": "$(AspnetReleaseDateStamp)",
-      "4.7-ltsc2016-Wcf-DateStamp": "$(WcfReleaseDateStamp)",
-      "4.7.1-ltsc2016-Runtime-DateStamp": "$(RuntimeReleaseDateStamp)",
-      "4.7.1-ltsc2016-Aspnet-DateStamp": "$(AspnetReleaseDateStamp)",
-      "4.7.1-ltsc2016-Wcf-DateStamp": "$(WcfReleaseDateStamp)",
-      "4.7.2-ltsc2016-Runtime-DateStamp": "$(RuntimeReleaseDateStamp)",
-      "4.7.2-ltsc2016-Aspnet-DateStamp": "$(AspnetReleaseDateStamp)",
-      "4.7.2-ltsc2016-Wcf-DateStamp": "$(WcfReleaseDateStamp)",
+      "4.6.2-ltsc2016-Runtime-DateStamp": "$(ltsc2016-RuntimeReleaseDateStamp)",
+      "4.6.2-ltsc2016-Aspnet-DateStamp": "$(ltsc2016-AspnetReleaseDateStamp)",
+      "4.6.2-ltsc2016-Wcf-DateStamp": "$(ltsc2016-WcfReleaseDateStamp)",
+      "4.7-ltsc2016-Runtime-DateStamp": "$(ltsc2016-RuntimeReleaseDateStamp)",
+      "4.7-ltsc2016-Aspnet-DateStamp": "$(ltsc2016-AspnetReleaseDateStamp)",
+      "4.7-ltsc2016-Wcf-DateStamp": "$(ltsc2016-WcfReleaseDateStamp)",
+      "4.7.1-ltsc2016-Runtime-DateStamp": "$(ltsc2016-RuntimeReleaseDateStamp)",
+      "4.7.1-ltsc2016-Aspnet-DateStamp": "$(ltsc2016-AspnetReleaseDateStamp)",
+      "4.7.1-ltsc2016-Wcf-DateStamp": "$(ltsc2016-WcfReleaseDateStamp)",
+      "4.7.2-ltsc2016-Runtime-DateStamp": "$(ltsc2016-RuntimeReleaseDateStamp)",
+      "4.7.2-ltsc2016-Aspnet-DateStamp": "$(ltsc2016-AspnetReleaseDateStamp)",
+      "4.7.2-ltsc2016-Wcf-DateStamp": "$(ltsc2016-WcfReleaseDateStamp)",
       "4.7.2-ltsc2019-Runtime-DateStamp": "$(RuntimeReleaseDateStamp)",
       "4.7.2-ltsc2019-Aspnet-DateStamp": "$(AspnetReleaseDateStamp)",
       "4.7.2-ltsc2019-Wcf-DateStamp": "$(WcfReleaseDateStamp)",
-      "4.8-ltsc2016-Runtime-DateStamp": "$(RuntimeReleaseDateStamp)",
-      "4.8-ltsc2016-Aspnet-DateStamp": "$(AspnetReleaseDateStamp)",
-      "4.8-ltsc2016-Wcf-DateStamp": "$(WcfReleaseDateStamp)",
-      "4.8-ltsc2016-Sdk-DateStamp": "$(SdkReleaseDateStamp)",
+      "4.8-ltsc2016-Runtime-DateStamp": "$(ltsc2016-RuntimeReleaseDateStamp)",
+      "4.8-ltsc2016-Aspnet-DateStamp": "$(ltsc2016-AspnetReleaseDateStamp)",
+      "4.8-ltsc2016-Wcf-DateStamp": "$(ltsc2016-WcfReleaseDateStamp)",
+      "4.8-ltsc2016-Sdk-DateStamp": "$(ltsc2016-SdkReleaseDateStamp)",
```

Comment:
This should be reverted. We don't install the new KB in the 4.8 Dockerfiles.

Reply:
I updated it such that only 3.5 LTSC 2016 images have the new datestamp.

Reply:
Now it's not updating enough. There are still other non-3.5 versions which are being updated to reference the new KB number. All of those versions should have an updated datestamp.

Reply:
OK - Tried to make sure it matches the Dockerflies that were changed this time. I added 4.7, 4.7.1, and 4.7.2.